crates/spark/src/tree/mod.rs

```diff
@@ -19,7 +19,8 @@ use uuid::Uuid;
 pub struct Leaves {
     pub available: Vec<TreeNode>,
     pub not_available: Vec<TreeNode>,
-    pub missing_from_operators: Vec<TreeNode>,
+    pub available_missing_from_operators: Vec<TreeNode>,
+    pub not_available_missing_from_operators: Vec<TreeNode>,
```

Comment:
Perhaps we can omit this field? We don't use it and I know spark are on their way to fix the "missing from operators" issue soon.

Reply:
Done in [8d32e1d](https://github.com/breez/spark-sdk/pull/319/commits/8d32e1dfb84e308ca79a705141d097fc5221367d)